refract-spec.md

```diff
@@ -36,18 +36,21 @@ The Refract Element contains four properties: `element`, `meta`, `attributes`, a
           - `ref` (Link) - Link to referenced element or type
           - `class` (array[string]) - Array of classifications for given element
           - `prefix` (string) - Prefix in which element MAY be found
+          - `name` (enum) - For use when an element is used as a property of an Object Element
+              - string
+              - Element
           - `namespaces` (array[Link]) - Include elements from given namespaces or prefix elements from given namespace
           - `title` (string) - Human-readable title of element
           - `description` (string) - Human-readable description of element
-      - (array[Member Element])
+      - (array[Property Element])
 
 - `attributes` (enum)
 
-    The `attributes` property defines attributes about the given instance of the element, as specified by the `element` property.
+    The `attributes` property defines attributes about the given instance of the element, as specified by the `element` property. When `attributes` is an object, it MAY contain elements itself. The element definition SHOULD be used when interacting with `attributes` and its properites and values.
```

Comment:
Good question—this needs clarity. This is stating that when interacting with the attributes, you should rely on the definition of the element so you know the types and values of each property. This way you know if something is refracted or not rather than introspecting and trying to figure it out.